src/Module/Api/Friendica/Events/Index.php

```diff
@@ -0,0 +1,75 @@
+<?php
+/**
+ * @copyright Copyright (C) 2020, Friendica
+ *
+ * @license   GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+namespace Friendica\Module\Api\Friendica\Events;
+
+use Friendica\Content\Text\BBCode;
+use Friendica\Database\DBA;
+use Friendica\Module\BaseApi;
+use Friendica\Network\HTTPException;
+
+/**
+ * api/friendica/events
+ *
+ * @package Friendica\Module\Api\Friendica\Events
+ */
+class Index extends BaseApi
+{
+	public static function rawContent(array $parameters = [])
+	{
+		if (self::login() === false) {
+			throw new HTTPException\ForbiddenException();
+		}
+
+		$since_id = $_REQUEST['since_id'] ?? 0;
+		$count    = $_REQUEST['count'] ?? 20;
+
+		$condition = ["`id` > ? AND `uid` = ?", $since_id, self::$current_user_id];
+		$params = ['limit' => $count];
+		$events = DBA::selectToArray('event', [], $condition, $params);
+
+		$items = [];
+		foreach ($events as $event) {
+			$items[] = [
+				'id'        => intval($event['id']),
```

Comment:
Nitpicker mode:
Shouldn't this be a class?

Reply:
Yes, but I was pressed by time since Marco is working on his app. I can't sit for too many hours coding today.